src/sjcl-custom/sjcl-ecdsa-signdeterministic.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+let sjcl = require('sjcl');
+/* eslint-disable new-cap */
+
+sjcl.ecc.ecdsa.secretKey.prototype.generateK =
+function(hash, hashObject) {
+
+  let curve = this._curve;
```

Comment:
I think it's fine to leave the let/const in and just replace with var when preparing the pull request to sjcl.